includes/bp-xprofile/classes/class-bp-rest-xprofile-field-groups-endpoint.php

```diff
@@ -861,6 +866,15 @@ public function get_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
+		$params['include_groups'] = array(
```

Comment:
`include` would be more in line with other endpoints. And since this is a field group endpoint, that might already be implied.

```suggestion
		$params['include'] = array(
```

Reply:
A bit below, there's the `exclude_groups` parameter, that's why I've used `include_groups` here. Just `include` is fine with me. I'll use this. Thanks for your suggestion.

Reply:
I see! In that case, `include_groups` would be a better option to keep it consistent.